app/src/lib/ui/ProNotebookFocus/ProNotebookFocusDetails.svelte

```diff
@@ -115,27 +115,21 @@
 		<div class="flex flex-col gap-4">
 			<h2 class="fr-h4 text-france-blue">Objectifs</h2>
 			<div>
-				<Accordions>
-					{#each targets as target (target.id)}
-						<Accordion
-							title={'<span>' +
-								target.target +
-								' - <em>' +
-								statusValues.find((value) => value.name == target.status)?.label +
-								'</em></span>'}
-						>
-							<Select
-								selectLabel={"Statut global de l'objectif"}
-								options={statusValues}
-								selected={target.status}
-								on:select={(event) => onChangeTargetStatus(event, target.id)}
-							/>
-							<ProNotebookActionList {target} theme={focus.theme} />
-						</Accordion>
-					{:else}
-						<p>Aucun objectif n'a été créé pour cet axe de travail.</p>
-					{/each}
-				</Accordions>
+				{#each targets as target (target.id)}
+					<span>
+						{target.target} -
+						<em> statusValues.find((value) => value.name == target.status)?.label </em></span
```

Comment:
cela devrait etre dans une `{}`
```suggestion
						<em>{statusValues.find((value) => value.name == target.status)?.label}</em></span
```